File: src/main/java/com/revolut/interview/model/request/TransferRequest.java
package com.revolut.interview.model.request;

import com.fasterxml.jackson.annotation.JsonAlias;
import lombok.ToString;
import lombok.Value;

/**
 * @author Ivan Zemlyanskiy
 */
@Value
@ToString
public class TransferRequest {

    @JsonAlias("donor")
    long donorId;
    @JsonAlias("acceptor")
    long acceptorId;
    String amount;

}

File: src/main/java/com/revolut/interview/model/request/TransferRequest.kt
package com.revolut.interview.model.request

import com.fasterxml.jackson.annotation.JsonAlias
import java.math.BigDecimal

/**
 * @author Ivan Zemlyanskiy
 */
class TransferRequest(@JsonAlias("acceptor")
                      val acceptorId: Long,
                      val amount: BigDecimal)
